### PR TITLE
Responsive web

### PR DESCRIPTION
**Pull request by promise-J**
First draft of the second project (responsive web design using HTML5 and CSS3)

### CODE REVIEW

**Comment by craigs40** [comment]
We would like to request that the TSE provide help in regards to our stylelint error: 

stylelinterror

**Comment by descholar-ceo** [comment]
# This Pull Request is under review by [descholar-ceo](https://github.com/descholar-ceo) :eyes: :eyes: 
## Please wait a bit ...

File: style.css [hunk 1]
```diff
@@ -0,0 +1,1077 @@
...
+.tnw-header .row1 {
+  display: flex;
+  align-items: flex-end;
+}
+
+.tnw-header .row1 {
+  /* display: flex; */
+
+  /* flex-direction: column; */
+}
```
Comment (descholar-ceo) [inline style.css]: Remove the commented codes

File: style.css [hunk 2]
```diff
@@ -0,0 +1,1077 @@
...
+  margin: auto;
+}
+
+.latest-news {
+  height: 200px;
+  width: 220px;
+}
+
+.latest-news h2 {
+  <<<<<<< HEAD
```
Comment (descholar-ceo) [inline style.css]: Remove this ` <<<<<<< HEAD`

File: style.css [hunk 3]
```diff
@@ -0,0 +1,1077 @@
...
+  color: black;
+  font-size: 17px;
+  font-weight: 900;
+  margin: 0.5rem;
+  =======
```
Comment (descholar-ceo) [inline style.css]: And fix this `=======`

File: style.css [hunk 4]
```diff
@@ -0,0 +1,1077 @@
...
+ color: black;
+  font-size: 13px;
+  font-weight: 800;
+  margin: 0.5rem;
+>>>>>>> ed941b3d8ef9a5158498657cd2b1130dbd51d065
```
Comment (descholar-ceo) [inline style.css]: And this should be removed `>>>>>>> ed941b3d8ef9a5158498657cd2b1130dbd51d065`

File: style.css [hunk 5]
```diff
@@ -0,0 +1,1077 @@
...
+
+  .tnw-header .tnw-grid1 {
+    grid-row: 1/2;
+    grid-column: 1/2;
+
+    /* justify-content: center; */
+
+    /* height: 50%; */
+  }
+
```
Comment (descholar-ceo) [inline style.css]: This commented code should be removed as well

File: style.css [hunk 6]
```diff
@@ -0,0 +1,1077 @@
...
+    padding: 0.5rem;
+  }
+
+  .section4 .latest-deal .latest-info {
+    height: 60px;
+    margin: auto;
+    justify-content: space-around;
+  }
+
+  <<<<<<< head ======= >>>>>>> ed941b3d8ef9a5158498657cd2b1130dbd51d065 .section4 .latest-deal .latest-deal-img {
```
Comment (descholar-ceo) [inline style.css]: This also should be removed

File: style.css [hunk 7]
```diff
@@ -0,0 +1,1077 @@
...
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+
```
Comment (descholar-ceo) [inline style.css:112]: [Optional] For the reset, it's better to do it in a separate file, to increase code readability and maintainability as well

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Changes Required :vertical_traffic_light: :warning: 
## Hello @promise-J and @craigs40  :handshake: :handshake: :handshake: 

:+1: This is good to make it this far, your clone looks like the original page on large screens and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have used the sementic html tags, keep it up, and your project is really almost done.
#### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
### Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 
### General overview and beaviours
### ===============================
- [ ] TWN logo appears at the leftmost part of the navbar on medium screens
- [ ] Add more description of the project on readme me file
- [ ] Work again on this Pull Request description => A recommended way of writting better Pull Request description is to answer the following questions in your description: What does this Pull Request do? and Which tasks to complete on this Pull Request?
- [ ] The page content should fit and cover the whole screen on large screens, learn the following screenshots to understand what you are required to change (screenshots on 904px screen)

| Your clone | Original page |
| --- | --- |
| Screenshot from 2021-01-28 17-33-07|Screenshot from 2021-01-28 17-33-58 |

:point_right: Could you see the difference :question: :question: :question: 

- [ ] Work again on mobile version, the mobile version seems to have a lot of works, compare the original website with you version, it doesn't have any resemblace

| Your clone | Original page |
| --- | --- |
| Screenshot from 2021-01-28 17-30-07 | Screenshot from 2021-01-28 17-31-05|

 => I took this screenshot on 320px of width: mobile screen
:point_right: Could you see the difference :question: :question: :question: 

- [ ] Fix the lates deal section on medium screens  => The articles should have margins between them and also remember to add the `See all deals` link with the `Latest Deals` link, they should be on the same line

| Your clone | Original page |
| --- | --- |
| Screenshot from 2021-01-28 17-26-04 | Screenshot from 2021-01-28 17-26-57 |

:point_right: Could you see the difference :question: :question: :question: 

- [ ] Fix the Categories (Apps, Gear, Tech, etc) section => The articles should have margins between them

### Codes
### ======
- [ ] It’s better to delete commented piece of codes. If you need to review a previous version of your code, use git history [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from the [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)
- [ ] Linters check has detected a lot of errors from your CSS codes, please fix them
    :warning: You can fix the issue with github actions by removing `node_modules` folder and `package-lock.json` file from the root directory of your project and run again this command `npm install --save-dev stylelint@13.3.x stylelint-scss@3.17.x stylelint-config-standard@20.0.x stylelint-csstree-validator`
:warning: This works, as you can see on [this PR](https://github.com/descholar-ceo/check-linters/pull/1) I fixed your issue, and [these are the errors](https://github.com/descholar-ceo/check-linters/pull/1/checks?check_run_id=1785016981) your CSS codes have. So try that as well, I hope it work for you as well!
- [ ] Ensure that all of `alt` attributes of the ` If you fix this issue it will increase your Search engine optimization (SEO) 

___
_If anything seems to be unclear to you, reachout and ask anything by commenting on this Pull Request and mention me as [descholar-ceo](https://github.com/descholar-ceo), I will receive a notification and will respond._

[GitHub](https://github.com/descholar-ceo) | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

_Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)_

**Comment by promise-J** [comment]
Implementation of the following action:
styleLints errors
removed all comments from the code
ensured the two break points for the medium screen and mobile screen are responsive
added more description of the project

File: index.html [hunk 8]
```diff
@@ -0,0 +1,584 @@
...
+        </div>
+      </div>
+    </div>
+    <main class="body">
+      <h1>Latest news
+        <i class="fa fa-angle-right" aria-hidden="true"></i>
+      </h1>
+      <section class="section1">
+        <div class="latest-news">
+          <img class="section1img" src="https://images.unsplash.com/photo-1494548162494-384bba4ab999?ixid=MXwxMjA3fDB8MHxzZWFyY2h8MXx8c3VucmlzZXxlbnwwfHwwfA%3D%3D&ixlib=rb-1.2.1&w=1000&q=80" alt=""><br>
```
Comment (umairarshadbutt) [inline index.html]: Please add the description in the `alt` attribute

File: style.css [hunk 9]
```diff
@@ -0,0 +1,1024 @@
...
+.section3 .item-set > ul {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+
+  /* background-color: blue; */
+
+  /* margin-top: 10px; */
```
Comment (umairarshadbutt) [inline style.css]: Remove the commented CSS

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @promise-J 
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [x] Remove commented CSS from css file. ([Article for support](https://speckyboy.com/good-bad-css-practices/)) ( [screenshot](https://user-images.githubusercontent.com/29689273/93404399-8997b500-f8a3-11ea-8f88-fb12b8f6ca80.png) from Pathwright)
- [x] Please add the description in the `alt` attribute (There are 19 places to be updated)
- [x] Please add the screenshot in the README file
- [x] Please update the updated live link in the README file
##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by layzrd — APPROVED** [review]
### Status: Approved 🙌 🎇
Hi @promise-J, I am  [Abdulaziz Redi Ali](https://azizali.ml/), your TSE for this PR.  Congratulations 🎊 👏, you did very well on this project.
Kindly fix the following issue. NB. It's an optional change.

- [ ] Fix the broken screenshot link in the readme.

Now, you can move ahead.

Happy coding 👨‍💻
[Abdulaziz Redi Ali](https://azizali.ml/) | [Contact me on Slack](https://microverse-students.slack.com/team/U011SD6JUH2)

**Pull request merged**